### ISSUE TITLE
Possibility to block access to a client/site

### ISSUE THREAD

**Issue by ShiryokuHorus** [amidaware/tacticalrmm#2052]
Hi, 
First of all thank you very much for existing. We are starting to use TacticalRMM in my company and it is really great !

I have an "issue" though, i'd like to restrict access to a site to a role and i don't find an easy way of doing so, maybe i'm doing something wrong. I'll explain myself:

I have to roles defined (Admin, Tech)
By default those two roles have access to all client (No client selected, implicit access to every client)

But i have one specific client that i want to restrict access to one user only (Or may be one role).
The way roles works it seems that you can only allow certain site, and not block. Meaning that i could set my roles to allow every client expect the client i want to block access to, but it may be hard to manage when adding new client not to forget to add the client to the roles.

Is there a way to block access to a specific client instead of allowing certain client by choosing them ?

Sorry if its not clear.. !

**Comment by dinger1986** [comment]
Yeah unfortunately it is setup to allow not block, maybe theres a way the devs can allow that to be inverted, you could open an issue as a feature request?

**Comment by ShiryokuHorus** [comment]
Yes for sure, thanks for reply
